src/changelog/.2.x.x/3874_remove_jvmrunargs_lookup.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<entry xmlns="https://logging.apache.org/xml/ns"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="
+           https://logging.apache.org/xml/ns
+           https://logging.apache.org/xml/ns/log4j-changelog-0.xsd"
+       type="removed">
+    <issue id="3874" link="https://github.com/apache/logging-log4j2/issues/3874"/>
+    <description format="asciidoc">
+      Removed the `jvmrunargs` lookup and related code due to lack of clear use case and potential for confusion.
```

Comment:
We don't need to elaborate on our reasoning in the changelog, interested people can check the linked issue:

```suggestion
      Remove the `jvmrunargs` lookup
```